listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -232,11 +234,13 @@ def submit_listens_to_listenbrainz(user: Dict, listens: List, listen_type=LISTEN
         listen_type: the type of listen (single, import, playing_now)
     """
     username = user['musicbrainz_id']
+    # insert payload looks for 'id' key whereas the user dict here has it in 'user_id' key
```

Comment:
Is this comment right? in the next line you use `user['user_id']` but the comment says `'id'`. id is used in the api endpoint.

Reply:
Yeah, the comment is accurate for the old code that did user['id'] = user['user_id'] but no longer needed that we now have the type. Will remove the comment.

Reply:
Fixed.